swift_browser_ui/api.py

```diff
@@ -100,6 +100,56 @@ async def swift_create_container(request: aiohttp.web.Request) -> aiohttp.web.Re
     raise aiohttp.web.HTTPClientError(reason=res["error"])
 
 
+async def swift_delete_container(request: aiohttp.web.Request) -> aiohttp.web.Response:
+    """Delete an empty container."""
```

Comment:
Well, the API can delete a non-empty container as well, the check is in UI side. Not allowing non-empty deletion was intentional on my part, since there's no versioning, undo-buttons, trashcans, any other way of recovering this – deleting a full container comes with a too large possibility for a mistake in my opinion. So for this reason I chose to require deleting all the objects inside the container first.